backends/aws-sqs/relay.go

```diff
@@ -99,6 +72,15 @@ func (r *Relayer) Relay() error {
 	// TODO: Optionally print out relay and SQS config
 
 	for {
+		select {
+		case <-r.ShutdownContext.Done():
+			r.log.Info("Received shutdown signal, existing relayer")
+			return nil
+		default:
+			// noop
+		}
+
```

Comment:
It "blocks" for as long as RelayWaitTimeSeconds is set - it's basically a poll.

Reply:
👍